frontend/src/components/InfoBox/index.js

```diff
@@ -253,36 +280,35 @@ class InfoBox extends Component {
 
           {pageType === 'department' && (
             <div className="department">
-              <div className="title">{this.props.data.name}</div>
-              <p className="subtitle">{this.props.data.code}</p>
+              <div className="title">{name}</div>
+              <p className="subtitle">{code}</p>
             </div>
           )}
         </div>
 
         {pageType !== 'department' && (
           <div id="banner-score">
-            <ScoreboxRow
+            <Ratings
               value="Average"
-              instructor={this.props.data.average_ratings.rInstructorQuality}
-              course={this.props.data.average_ratings.rCourseQuality}
-              difficulty={this.props.data.average_ratings.rDifficulty}
-              num_sections={this.props.data.num_sections}
+              instructor={avgInstructorQuality}
+              course={avgCourseQuality}
+              difficulty={avgDifficulty}
+              num_sections={numSections}
             />
 
-            <ScoreboxRow
+            <Ratings
               value="Recent"
-              instructor={this.props.data.recent_ratings.rInstructorQuality}
-              course={this.props.data.recent_ratings.rCourseQuality}
-              difficulty={this.props.data.recent_ratings.rDifficulty}
-              num_sections={this.props.data.num_sections_recent}
+              instructor={rcntInstructorQuality}
+              course={rcntCourseQuality}
+              difficulty={rcntDifficulty}
```

Comment:
would prefer just writing out `recent` since it doesn't save that many chars lol